{
  "model": {
    "shape": "(1,100)",
    "bounds" : "[(0,1)]",
    "layers": [
      {
        "type": "embedding",
        "input_dim": "32187",
        "output_dim": "100",
        "weight_shape":"(32186,100)"
      },
      {
        "type": "Conv1D",
        "weights": "",
        "bias": "",
        "nodes": "64",
        "kernel_size": "(7)",
        "func": "relu",
        "padding": "true"
      },
      {
        "type": "MaxPool1D",
        "size": "(2)"
      },
      {
        "type": "Conv1D",
        "weights": "",
        "bias": "",
        "nodes": "64",
        "kernel_size": "(7)",
        "func": "relu",
        "padding": "true"
      },
      {
        "type": "GlobalMaxPool1D"
      },
      {
        "type": "dropout",
        "rate": "0.5"
      },
      {
        "type": "linear",
        "weights": "",
        "bias": "",
        "nodes": "32",
        "func": "relu"
      },
      {
        "type": "linear",
        "weights": "",
        "bias": "",
        "nodes": "10",
        "func": "softmax"
      }
    ]
  },
  "source": "https://stackoverflow.com/questions/58548520/tensorflow-keras-embedding-layer-error-layer-weight-shape-not-compatible"
}
